Update common-questions.mdx (#8065)
PostHog/posthog.com: contents/docs/feature-flags/common-questions.mdx

To ensure they experience consistent behavior, check that you call `identify()` before evaluating the feature flag.

> > *Note*: This does not apply to server-side SDKs, because you're explicitly telling us which distinct ID to use to evaluate flags, rather than depending on the identify() event to tell us which distinct ID to use.

Reviewer: ```suggestion
> *Note*: This only applies to frontend SDKs. Server-side SDKs do not need to call `identify()` ```

> > *Note*: This does not apply to server-side SDKs, because you're explicitly telling us which distinct ID to use to evaluate flags, rather than depending on the identify() event to tell us which distinct ID to use.

Reviewer 2: Looks good. I would avoid stating the claim about identify on server so strongly for now @Lior539  because, while somewhat rare, we _do_ explicitly tell users to call `identify` from the backend for a PoE v2 workaround (when signup event is on backend, not properly merged with anon frontend events unless identify is called on server...).

Reviewer: Thanks Cameeron. My understanding is that the scenario you described above, they should use `alias` right? and not identify? (I know alias uses identify under the hood though).

>  I would avoid stating the claim about identify on server so strongly for now

This approach is also aligned with how we described it in the [identify docs](https://posthog.com/docs/product-analytics/identify) i.e. something you use on the frontend + we only show frontend code snippets
